spec/models/alchemy/page_spec.rb

```diff
@@ -181,48 +181,6 @@ module Alchemy
             expect(page.legacy_urls).to be_empty
           end
         end
-
-        context "page is public" do
-          before do
-            expect(page).to receive(:public?) { true }
-          end
-
-          context "and published_at is nil" do
-            before do
-              expect(page).to receive(:published_at) { nil }
-            end
-
-            it "should set published_at" do
-              expect {
-                page.save!
```

Comment:
These tests need to stay, because we want to values to change if someone enters a `public_on` date in the page settings overlay.